variable/economy/economy.yaml

```diff
@@ -4,6 +4,18 @@ Consumption:
    description: Total consumption of all goods, by all consumers in a region
    unit: billion US$2010/yr
 
+Consumption|Electricity:
+   description: Total consumption of electricity, by all consumers in a region
+   unit: billion US$2010/yr
+   
+Consumption|Percentage change relative to 2020:
```

Comment:
The percentage change is clearly a derived measure, so this variable is duplicating information - it would be more rigorous to only report absolute quantities and let users compute relate changes themselves.

Reply:
I only wanted to add a measure of the change of physical quantities, but it does not make sense when we consider the aggregated consumption. Therefore I agree with your comment and leave it as a monetary flow